test-d/formio/components/content.test-d.ts

```diff
@@ -38,19 +38,19 @@ expectAssignable<ContentSchema>({
 });
 
 // Bad customClass value
-expectNotAssignable<ContentSchema>({
+expectNotAssignable<ContentComponentSchema>({
   id: 'eqegfc',
   type: 'content',
   html: '',
   key: 'content',
   customClass: 'arbitraryValue',
-});
+} as const);
```

Comment:
What does this do?

Reply:
it treats the entire object as a literal/constant, instead of inferring wider types from it - otherwise it came up with `customClass` being `string` instead of the literal `arbitraryValue`.

This caused some weird funky bugs (`expectNotAssignable` is a footgun) where tests were passing despite the actual object not being valid if you were to use the `ContentComponentSchema` for the variable. With the `as const`, these tests now actually fail if it turns out to be assignable despite not being expected to.